cli.js

```diff
@@ -1,26 +1,34 @@
 #!/usr/bin/env node
 const fs = require('fs');
 const { argv } = require('yargs')
-  .command('* <file-path>', 'parse and convert the input file', (yargs) => {
-    yargs.positional('file-path', {
-      describe: 'the input USFM or JSON file to be parsed and converted',
+  .command('* <file>', 'Parse/validate USFM 3.x to/from JSON.', (yargs) => {
```

Comment:
Here we can give a USFM or JSON file as input. But only parsing operation will be done on it, not validate. If the file is not valid, it throws an error(not True/False, like in validate() method). Does this convey that correctly?

Reply:
So, the output right now would be:
```
usfm-grammar <file>

Parse/validate USFM 3.x to/from JSON.

Positionals:
  file  The path of the USFM or JSON file to be parsed and/or converted. By
        default, auto-detects input USFM and converts it into JSON and
        vice-versa.

Options:
  -l, --level    Level of strictness in parsing. This defaults to `strict`.
                                                            [choices: "relaxed"]
      --filter   Filter out content from input USFM. Not applicable for input
                 JSON or for CSV/TSV output.              [choices: "scripture"]
  -o, --output   The output format to convert input into.
                                         [choices: "csv", "tsv", "usfm", "json"]
  -h, --help     Show help                                             [boolean]
  -v, --version  Show version number                                   [boolean]
```

Here, the string `Parse/validate USFM 3.x to/from JSON.` is a one line description of the tool itself. This is similar with how you would see the help text using `man` in GNU/Linux. More details can then be found on further reading or via the README.